Compile failed because of linking to libatomic.so on aarch64

Linking  libatomic.so is required only for 32-bit ARMR operating systems，but redundant on 64-bit systems. If libatomic.so not provided on 64-bit systems, it will lead to compilie failure.

**Crash report**
    LINK redis-server
/usr/bin/ld: cannot find -latomic
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:228: redis-server] Error 1
make[1]: Leaving directory '/usr1/jenkins/workspace/FenixMainService/ipsec/open_source/redis/redis-5.0.8/src'
make: *** [Makefile:6: all] Error 2
```
```

**Aditional information**
1. OS distribution and version
 Linux 7.222.103.233 4.19.90-vhulk2009.2.0.h269.eulerosv2r9.aarch64 #1 SMP Sun Sep 27 12:47:01 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux



![image](https://user-images.githubusercontent.com/49217237/115645245-9d802980-a352-11eb-9dad-986cb5ea669b.png)

--- a/src/Makefile
+++ b/src/Makefile
@@ -93,13 +93,9 @@ FINAL_LDFLAGS=$(LDFLAGS) $(REDIS_LDFLAGS) $(DEBUG)
 FINAL_LIBS=-lm
 DEBUG=-g -ggdb
 
-# Linux ARM needs -latomic at linking time
-ifneq (,$(filter aarch64 armv,$(uname_M)))
-        FINAL_LIBS+=-latomic
-else
+# Linux ARM32 needs -latomic at linking time
 ifneq (,$(findstring armv,$(uname_M)))
         FINAL_LIBS+=-latomic
-endif
 endif
 
 ifeq ($(uname_S),SunOS)

